script/s3.js

```diff
@@ -171,6 +198,8 @@ const uploadFiles = () => recursiveRead(buildPath, (err, results) => {
         }
       );
     }
+    deployTime = (new Date().getTime() - start.getTime()) / 1000;
+    persistDeployTime(deployTime)
```

Comment:
I'd check the presence of the metrics object here like:
```suggestion
if (awsCredentials.metrics) {
    persistDeployTime(deployTime)
}
```